linode/instance/schema_datasource.go

```diff
@@ -406,4 +408,38 @@ var instanceDataSourceSchema = map[string]*schema.Schema{
 			},
 		},
 	},
+	"placement_group": {
+		Type:     schema.TypeList,
+		Computed: true,
+		Optional: true,
```

Comment:
We probably don't need to put Optional here, since it's only for read.

Reply:
Fixed!